public/index.html

```diff
@@ -5,6 +5,7 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
     <link rel="stylesheet" href="%PUBLIC_URL%/cuisine-webfont/stylesheet.css" />
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css" />
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAbWsYd03RL-T_1VZW5wU7Zod2u9H3QIXM&language=en&region=USA"></script>
```

Comment:
You should prefer using an environment variable here.
```suggestion
    <script src="https://maps.googleapis.com/maps/api/js?key=%GOOGLE_API_KEY%&language=en&region=USA"></script>
```
Hope i'm not mistaken with the syntax.